python/action-server/test_statechart.py

```diff
@@ -0,0 +1,23 @@
+import json
+import yaml
+from state_machine import StateMachine
+import main
+
+def test_statechart():
+    with open('action.yml') as file:
+        try:
+            settingConfig = yaml.safe_load(file)  
+        except yaml.YAMLError as err:
+            print(err)
+
+    settings = main.ActionSettings(**settingConfig)
```

Comment:
Define a pytest test fixture to have all this setup code required to run these test